src/prefect_server/database/models.py

```diff
@@ -307,6 +307,7 @@ class FlowGroup(HasuraModel):
     id: UUIDString = None
     created: datetime.datetime = None
     updated: datetime.datetime = None
+    description: str = None
```

Comment:
I'm confused that it's a string here but a json blob in the database and graphql schema

Reply:
You are correct to be confused - it should be a string everywhere; will update!

Reply:
Updated